.travis.yml

```diff
@@ -8,10 +8,25 @@ cache:
   directories:
     - $HOME/.ivy2
 env:
-  - HADOOP_VERSION="1.0.4"
-  - HADOOP_VERSION="1.2.1"
-  - HADOOP_VERSION="2.2.0"
+  matrix:
+    - HADOOP_VERSION="1.0.4"
+    - HADOOP_VERSION="1.2.1"
+    - HADOOP_VERSION="2.2.0"
+  global:
+    # AWS_REDSHIFT_JDBC_URL
+    - secure: "RNkxdKcaKEYuJqxli8naazp42qO5/pgueIzs+J5rHwl39jcBvJMgW3DX8kT7duzdoBb/qrolj/ttbQ3l/30P45+djn0BEwcJMX7G/FGpZYD23yd03qeq7sOKPQl2Ni/OBttYHJMah5rI6aPmAysBZMQO7Wijdenb/RUiU2YcZp0="
+    # AWS_REDSHIFT_PASSWORD
+    - secure: "Bzre/ohanBt6wrj5asn8+iaIU5qm2QBZ+P/PiAeg55R5sqfyI/pwCYZKdtKSG7SuKzsoiAOtnjvcXMD2hickTLIDz3GmrvFcpx7yn3PEKoLQfT4Ry1/RMOsqa1+sj6zJ7J2dl4w0AURJ7Jb9/7GRylNnL0jkUvqUnWet8PBb7R8="
+    # AWS_REDSHIFT_USER
+    - secure: "LIkY/ZpBXK3vSFsdpBSRXEsgfD2wDF52X8OZOlyBJOiZpS4y1/obj8b3VQABDPyPH95bGX/LOpM0vVM137rYgF0pskgVEzLMyZOPpwYqNGPf/d4BtQhBRc8f7+jmr6D4Hrox4jCl0cCKaeiTazun2+Y9E+zgCUDvQ8y9qGctR2k="
+    # AWS_ACCESS_KEY_ID
+    - secure: "CDlql+nrgdi7sUr7bYyXF4CFoOUCiJG9WEYNRV4k/lC37eS/al3iVYicnXqF+6UrPv5a4kHulG4d3g78J4hzn4ZVJuEhn6v8beoOBUoJJ7W/J05hVwGiQFxUq86wT3tIaBrAuDmOXaAnPEvDmPfJGNZL9ZG1CaQJo70R/HkbbVA="
+    # AWS_SECRET_ACCESS_KEY
```

Comment:
I wonder if I should purposely give these different names than the ones expected by the Amazon SDKs; this might be necessary in order to be able to test credential mechanisms (e.g. for writing a regression test for #32)